### ISSUE TITLE
AppNexus bid adapter no longer passes Criteo user ID

### ISSUE THREAD

**Issue by dbirkbeck** [prebid/Prebid.js#4860]
## Type of issue

- [x] Bug

## Description

The CriteoID user module replaced the CrteoRTUS user module with commit #4287 but the AppNexus bid adapter was not updated to pass the correct user ID value. (The appnexusBidAdapter.js still references criteortus.) As a result, the CriteoID user information is not passed.

## Steps to reproduce

Build the AppNexus bid adapter with the userId and CriteoId user modules.
Include the following in the Prebid configuration:
```
userSync: {
  userIds: [{ name: "criteo" }]
}
```
Load a test page in any browser.
Using the browser development tools, inspect the bid request and note the missing criteo eid in the user object within the HTTP request.

## Test page

Use the following page as a template to start from:
http://prebid.org/dev-docs/examples/basic-example.html

### Expected results
Within the AppNexus bid request there should be an object containing the Criteo ID. The object should be located at: user.ext.eids.[number] and should contain the following values:
source: "criteo.com"
uids: [{ _uid_ }]

### Actual results
No value is present

## Platform details

Prebid 3.7.0 (Possibly versions back to Nov 5, 2019)
All browsers

## Other information

The pull request that introduced the user module (CriteoID) that replaced (CriteoRTUS) is #4287.

**Comment by jsnellbaker** [comment]
@dbirkbeck thanks for reporting this, I'll update the adapter file.

**Comment by dbirkbeck** [comment]
I noticed that the CriteoRTUS module has now been removed in 3.8.0 but the PR to correct the Appnexus bid adapter has not been approved. This leaves the Appnexus adapter stranded with neither a working CriteoRTUS module nor a fix to allow the CriteoID module to work.

Any update on when the PR will be approved?

_This is the commit from the 3.8.0 release notes:_
Remove deprecated Criteo Rtus Module (#4835)
